scripts/SANS/isis_instrument.py

```diff
@@ -1725,8 +1730,14 @@ def is_run_new_style_run(workspace_ref):
         @param workspace_ref:: A handle to the workspace
         '''
         try:
-            run_num = workspace_ref.getRun().getLogData('run_number').value
+            # If we are dealing with a WorkspaceGroup then this will not contain any run number information,
+            # hence we need to access the first child workspace
+            if isinstance(workspace_ref, WorkspaceGroup):
+                run_num = workspace_ref[0].getRun().getLogData('run_number').value
+            else:
+                run_num = workspace_ref.getRun().getLogData('run_number').value
```

Comment:
Lines 1733 to 1738 look like a duplicate of lines 779 to 784. This could be handled by a function.